### PR TITLE
Add support for version 6xx

### PR DESCRIPTION
**Pull request by imbillow**
For existing idb file(6.2-8)

There is a series of sequential PR, this is 1/3

- 1/3 #91
- 2/3 #92
- 3/3 #93

### CODE REVIEW

File: tests/test_analysis.py [hunk 1]
```diff
@@ -633,43 +653,6 @@ def test_idainfo(kernel32_idb, version, bitness, expected):
...
-        if_exists(720, 64, None),
-        if_exists(730, 32, None),
-        if_exists(730, 64, None),
-        if_exists(740, 32, None),
-        if_exists(740, 64, None),
-        if_exists(750, 32, None),
-        if_exists(750, 64, None),
-    ]
-)
-def test_idainfo720plus(kernel32_idb, version, bitness, expected):
```
Comment (XVilka) [inline tests/test_analysis.py:648]: Why remove this?

**Comment by imbillow** [comment]
It can be merged into test_idainfo.
In fact, test_idainfo now tests v6.2-v7.3.


Anton Kochkov <notifications@github.com> 于 2020年8月15日周六 上午2:00写道：

File: tests/test_analysis.py (hunk 1 above)
Comment (imbillow) [reply tests/test_analysis.py:648]: I think the `test_idainfo` is enough, this looks superfluous
https://github.com/williballenthin/python-idb/blob/dbbc2e7126207ef1b6410c37a6ef8bbf6adeb945/tests/test_analysis.py#L616-L653

**Comment by williballenthin** [comment]
this is great work

**Pull request merged**